account_payment_promissory_note_number/models/account_payment.py

```diff
@@ -0,0 +1,10 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields
+
+
+class AccountPayment(models.Model):
+    _inherit = "account.payment"
+
+    promissory_note_number = fields.Char(string='Promissory note number')
```

Comment:
string attribute is unnecessary, because Odoo will name it exactly as you have defined it. See https://www.odoo.com/documentation/15.0/developer/reference/backend/orm.html#odoo.fields.Field

![imagen](https://github.com/solvosci/slv-account/assets/12749832/6eb18bb2-cfec-43c9-9b41-b67c59718e72)